Cargo.toml

```
@@ -24,7 +24,7 @@ rand = "^0.7"
 sled = { version = "0.34", optional = true }
 electrum-client = { version = "0.8", optional = true }
 reqwest = { version = "0.11", optional = true, features = ["json"] }
-ureq = { version = "2.1", default-features = false, features = ["json"], optional = true }
+ureq = { version = "2.1", default-features = false, features = ["json", "tls"], optional = true }
```

Since `tls` the only default feature for ureq I'd change this line to:
```suggestion
ureq = { version = "2.1", features = ["json"], optional = true }
```
Also per below comment should be fine to have `tls` always enabled:
https://github.com/bitcoindevkit/bdk-cli/pull/41#discussion_r698333498

---

Noted. Will be done one a different PR.